test/OrchardCore.Tests/Modules/OrchardCore.Media/DefaultMediaFileStoreTests.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Moq;
+using OrchardCore.FileStorage;
+using OrchardCore.Media.Core;
+using OrchardCore.Media.Events;
+using Xunit;
+
+namespace OrchardCore.Tests.Modules.OrchardCore.Media
+{
+    public class DefaultMediaFileStoreTests
+    {
+        [Theory]
+        [InlineData("", "baz", "/media/baz")]
+        [InlineData("", "/media/baz", "/media/baz")]
+        [InlineData("bobs_your_uncle", "baz", "bobs_your_uncle/media/baz")]
+        [InlineData("bobs_your_uncle", "bobs_your_uncle/media/baz", "bobs_your_uncle/media/baz")]
```

Comment:
I can't understand from the tests who bob is, or which ones are supposed to be cdn paths, absolute paths, other paths unfortunately.